src/RobotSimulator/StdWorldBuilder.ts

```diff
@@ -74,6 +74,30 @@ export class StdWorldBuilder {
     }
 
     const robotSpec = robotBuilder.generateSpec();
+
+    robotSpec.mechanisms = [
+      {
+        type: "gripper-mechanism",
+        ioMap: [
+          {
+            id: "grab",
+            channel: 0,
```

Comment:
I think some other sensors might conflict with these values. but i might be wrong there.

Reply:
Checking now. I had assumed that they were different channels types, but will step through on the simulator core side. Worst comes to worst ill just bump it up to unique numbers

Reply:
Looks like they are considered separate systems, and don't conflict. Want me to increment them anyway so they're unique for the end user or just leave as is?